tutormfe_extensions/patches/openedx-cms-production-settings

```diff
@@ -1,5 +1,5 @@
 {%- if MFE_EXTENSIONS_BY_PATH %}
-{%- if get_mfe("course-authoring") %}
-COURSE_AUTHORING_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ CMS_HOST }}/course-authoring"
+{%- if get_mfe("authoring") %}
+COURSE_AUTHORING_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ CMS_HOST }}/authoring"
```

Comment:
Is it `authoring` or `course-authoring`

Reply:
look at https://github.com/overhangio/tutor-mfe/pull/225

Reply:
I mean the final path `/authoring`